### PR TITLE
Add Polars GPU engine page

### PR DESCRIPTION
**Pull request by exactlyallan**
Staging cudf polars page for review and content updates.

### CODE REVIEW

File: layouts/page/cudf-polars.html [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+          </code>
+        </div>
+      </div>
+
+    </div>
+  </div>
+</section>
+
+
+<!-- FAQ -->
```
Comment (singhmanas1) [inline layouts/page/cudf-polars.html:156]: Alan - Can we add FAQs in a sequential format with an accordian clickable format? This one seems a little tough to read right now. 


<section class="row section bg-rapids-gray padding-top-slant padding-bottom-slant">
  <div class="col">
    <div class="container">
      <h1 class="mb-4">Frequently Asked Questions</h1>
      
      
      <div class="accordion" id="faqAccordion">

        
        <div class="accordion-item">
          <h2 class="accordion-header" id="headingOne">
            <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
              Do I have to install any RAPIDS specific package for the GPU backend to work?
            </button>
          </h2>
          <div id="collapseOne" class="accordion-collapse collapse show" aria-labelledby="headingOne" data-bs-parent="#faqAccordion">
            <div class="accordion-body">
              No, you just need to install Polars with a feature flag in your standard Python pip install command: 
              <br><code>pip install polars[gpu]--extra-index-url=https://pypi.nvidia.com</code>
            </div>
          </div>
        </div>

        
        <div class="accordion-item">
          <h2 class="accordion-header" id="headingTwo">
            <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">
              Can I use the cuDF backend for Polars on my local Windows machine?
            </button>
          </h2>
          <div id="collapseTwo" class="accordion-collapse collapse" aria-labelledby="headingTwo" data-bs-parent="#faqAccordion">
            <div class="accordion-body">
              Yes, you can use the cuDF backend for Polars on a Windows machine by installing WSL2. Note: You would need WSL2 on a Windows 11 or above OS. For download and troubleshooting instructions, please visit <a href="#" target="_blank">RAPIDS installation</a> page.
            </div>
          </div>
        </div>

File: layouts/page/cudf-polars.html [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+ <div id="get-started"></div>
+<section class="row section padding-bottom-slant">
+  <div class="col">
+    <div class="container">
+
+      <h1 class="mb-4"> How It Works</h1>
+      <div class="row">
+        <div class="col-md-6 py-3">
+          <h2><i class="fa-sharp fa-light fa-gear-complex-code"></i> Default GPU </h2>
+          <p>First, install the cuDF backend for Polars with a feature flag in your standard Python pip install command:</p>
```
Comment (singhmanas1) [inline layouts/page/cudf-polars.html:92]: This step in common in both default and custom configurations. Any chance we could keep the pip installation command just right after "How it works" section? Something like https://docs.google.com/document/d/1h2Ynp0sniNuQ1k7Cfx64DYauaLJnadIkVpWCrXudaK0/edit?

File: layouts/page/cudf-polars.html [hunk 3]
```diff
@@ -235,31 +295,30 @@ <h2><i class="fa-sharp fa-light fa-gear-complex-code"></i> Share Feedback</h2>
...
+      labels: ['20GB', '40GB', '60GB'],
       datasets: [
         {
-          label: 'stand alone pandas',
+          label: '.collect()',
           barPercentage: 0.6,
-          data: [267, 274],
-          backgroundColor: 'rgb(33, 150, 243, 0.8)',
-          minBarLength: 4
+          data: [33, 65, 132],
```
Comment (singhmanas1) [inline layouts/page/cudf-polars.html:307]: [35,67,136]

File: layouts/page/cudf-polars.html [hunk 4]
```diff
@@ -235,31 +295,30 @@ <h2><i class="fa-sharp fa-light fa-gear-complex-code"></i> Share Feedback</h2>
...
+          backgroundColor: 'rgb(33, 150, 243, 0.8)'
         },
         {
-          label: 'cuDF pandas accelerator mode',
+          label: '.collect(engine="gpu")',
           barPercentage: 0.6,
-          data: [1, 2],
-          backgroundColor: 'rgb(115, 6, 255, 0.8)',
-          minBarLength: 4,
+          data: [12, 18, 32],
```
Comment (singhmanas1) [inline layouts/page/cudf-polars.html:313]: [12,19,32]

File: layouts/page/cudf-polars.html [hunk 5]
```diff
@@ -271,7 +330,81 @@ <h2><i class="fa-sharp fa-light fa-gear-complex-code"></i> Share Feedback</h2>
...
+  const ctx2 = document.getElementById('perfchartspeed');
+  new Chart(ctx2, {
+    type: 'bar',
+    data: {
+      labels: ['Q13', 'Q17', 'Q18', 'Q21'],
+      datasets: [
+        {
+          label: 'Polars (CPU)',
+          barPercentage: 0.6,
+          data: [7, 11, 17, 41],
```
Comment (singhmanas1) [inline layouts/page/cudf-polars.html:381]: [7,12,17,41]

File: layouts/page/cudf-polars.html [hunk 6]
```diff
@@ -271,7 +330,81 @@ <h2><i class="fa-sharp fa-light fa-gear-complex-code"></i> Share Feedback</h2>
...
+          backgroundColor: 'rgb(33, 150, 243, 0.8)'
+        },
+        {
+          label: 'Polars (GPU)',
+          barPercentage: 0.6,
+          data: [1, 2, 1, 4],
```
Comment (singhmanas1) [inline layouts/page/cudf-polars.html:387]: [1,2,1,3]

File: layouts/page/cudf-polars.html [hunk 7]
```diff
@@ -54,23 +50,24 @@ <h2 class="mt-4"><i class="fa-light fa-bolt-lightning"></i> Query Optimizer</h2>
...
-      <h1 class="mb-4"> RAPIDS cuDF is the go-to Polars backend for large datasets and compute heavy queries </h1>
+      <h1 class="mb-4"> The go-to Polars backend</h1>
       <div class="row">
-        <div class="col-md-6 py-3">
-          <h2><i class="fa-sharp fa-light fa-gear-complex-code"></i> Performance improves as <br> data scales </h2>
-          {{ with .Resources.GetMatch "polars-collect.png" }}
-          <img src="{{ .RelPermalink }}">
-          {{ end }}
+        <div class="col">
+          <p class="subheading">RAPIDS cuDF is the go-to Polars backend for large datasets and compute heavy queries.</p>
```
Comment (singhmanas1) [inline layouts/page/cudf-polars.html:56]: Don't need this sentence --> RAPIDS cuDF is the go-to Polars backend for large datasets and compute heavy queries

File: layouts/page/cudf-polars.html [hunk 8]
```diff
@@ -54,23 +50,24 @@ <h2 class="mt-4"><i class="fa-light fa-bolt-lightning"></i> Query Optimizer</h2>
   <div class="col">
     <div class="container">
 
-      <h1 class="mb-4"> RAPIDS cuDF is the go-to Polars backend for large datasets and compute heavy queries </h1>
+      <h1 class="mb-4"> The go-to Polars backend</h1>
```
Comment (singhmanas1) [inline layouts/page/cudf-polars.html:53]: Replace with 

PROCESS 100s OF MILLIONS OF ROWS IN SECONDS

File: layouts/page/cudf-polars.html [hunk 9]
```diff
@@ -80,30 +77,34 @@ <h2><i class="fa-light fa-diagram-subtask"></i>  Higher acceleration with Comput
...
             pip install polars[gpu] —extra-index-url=https://pypi.nvidia.com
           </code>
          
-          <br>
+        </div>
+      </div>
 
+      <div class="row">
+        <div class="col-md-6 py-6">
+          <h2><i class="fa-sharp fa-light fa-gear-complex-code"></i> Default GPU </h2>
```
Comment (singhmanas1) [inline layouts/page/cudf-polars.html:99]: Can we replace 'Default GPU' by 'Default GPU Backend'

File: layouts/page/cudf-polars.html [hunk 10]
```diff
@@ -113,30 +114,23 @@ <h2><i class="fa-sharp fa-light fa-gear-complex-code"></i> Default GPU </h2>
                   ).collect(engine="gpu"))
           </code>
         </div>
-        <div class="col-md-6 py-3">         
+        <div class="col-md-6 py-6">
```
Comment (singhmanas1) [inline layouts/page/cudf-polars.html:117]: Can we replace `Custom GPU` by `Custom GPU Backend`

File: layouts/page/cudf-polars.html [hunk 11]
```diff
@@ -0,0 +1,441 @@
...
+<div class="bottom-slant-container">
+  <div class="bottom-slant-down bg-white"></div>
+</div>
+
+<!-- Charts -->
+<section class="row section padding-top-slant">
+  <div class="col">
+    <div class="container">
+
+      <h1 class="text-center mb-4"> Process 100s of Millions of Rows in Seconds </h1>
```
Comment (singhmanas1) [inline layouts/page/cudf-polars.html:53]: 100s is appearing as 100S. If it defaults to all caps, can we use 'HUNDREDS'

PROCESS HUNDREDS OF MILLIONS OF ROWS IN SECONDS

File: layouts/page/cudf-polars.html [hunk 12]
```diff
@@ -0,0 +1,441 @@
...
+          }
+        }
+      },
+      plugins: {
+        legend: {
+          display: true
+        },
+        title: {
+          display: true,
+          text: 'PDS-H benchmark 80GB query processing time across a range of dataset sizes'
```
Comment (singhmanas1) [inline layouts/page/cudf-polars.html:352]: Can we say - 

'Query Processing time across a range of dataset sizes' 

This is not just for 80 GB

File: layouts/page/cudf-polars.html [hunk 13]
```diff
@@ -0,0 +1,441 @@
...
+<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/4.4.1/chart.umd.min.js"
+  integrity="sha512-CQBWl4fJHWbryGE+Pc7UAxWMUMNMWzWxF4SQo9CgkJIN1kx6djDQZjh3Y8SZ1d+6I+1zze6Z7kHXO7q3UyZAWw=="
+  crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+
+<script>
+  const ctx1 = document.getElementById('perfchartdata');
+  new Chart(ctx1, {
+    type: 'bar',
+    data: {
+      labels: ['20GB', '40GB', '60GB'],
```
Comment (singhmanas1) [inline layouts/page/cudf-polars.html:298]: This is 80GB

File: layouts/page/cudf-polars.html [hunk 14]
```diff
@@ -0,0 +1,441 @@
...
+      datasets: [
+        {
+          label: '.collect()',
```
Comment (singhmanas1) [inline layouts/page/cudf-polars.html:301]: Can we change this to 'Polars (CPU)'

File: layouts/page/cudf-polars.html [hunk 15]
```diff
@@ -0,0 +1,441 @@
...
+          barPercentage: 0.6,
+          data: [35,67,136], 
+          backgroundColor: 'rgb(33, 150, 243, 0.8)'
+        },
+        {
+          label: '.collect(engine="gpu")',
```
Comment (singhmanas1) [inline layouts/page/cudf-polars.html:307]: Can we change this to 'Polars (GPU)'

File: layouts/page/cudf-polars.html (hunk 13 above)
Comment (exactlyallan) [reply layouts/page/cudf-polars.html:298]: does anything need to change?

File: layouts/page/cudf-polars.html [hunk 16]
```diff
@@ -0,0 +1,441 @@
...
+          }
+        }
+      },
+      plugins: {
+        legend: {
+          display: true
+        },
+        title: {
+          display: true,
+          text: 'PDS-H benchmark 80GB top performing compute heavy queries with complex groupby and join operations'
```
Comment (singhmanas1) [inline layouts/page/cudf-polars.html:426]: Can we make this -

Top performing compute heavy queries with complex groupby and join operations (PDS-Benchmark; 80 GB dataset)

File: layouts/page/cudf-polars.html [hunk 17]
```diff
@@ -0,0 +1,441 @@
...
+          <canvas id="perfchartdata" width="500" height="375"></canvas>
+        </div>
+        <div class="col-md-6">
+          <h3 class="text-center"> Accelerate Polars Workflows up to 13x</h3>
+          <canvas id="perfchartspeed" width="500" height="375"></canvas>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col">
+          <p class="fw-lighter text-center">*GPU: NVIDIA H100, CPU: Intel Xeon W9-3495X (Sapphire Rapids), Storage: Local NVMe derived from TPC-H but these results are not comparable to TPC-H results <a href="https://github.com/pola-rs/polars-benchmark" target="_blank">Learn More <i class="fa-solid fa-arrow-up-right"></i></a></p>
```
Comment (singhmanas1) [inline layouts/page/cudf-polars.html:66]: Can we make this-

PDS-H benchmark | GPU: NVIDIA H100 | CPU: Intel Xeon W9-3495X (Sapphire Rapids) | Storage: Local NVMe <New line>
Note: PDS-H is derived from TPC-H but these results are not comparable to TPC-H results. [Learn More](https://github.com/pola-rs/polars-benchmark)

File: .devcontainer/devcontainer.json
Comment (bdice) [inline .devcontainer/devcontainer.json:1]: Let’s not delete the devcontainer in this PR. I have used it before to help build the site.

File: .gitignore [hunk 18]
```diff
@@ -6,3 +6,4 @@ _site
 public
 resources
 node_modules
+.devcontainer
```
Comment (bdice) [inline .gitignore:9]: This is not supposed to be ignored.

File: assets/scss/rapidsai/__general.scss [hunk 19]
```diff
@@ -204,6 +204,14 @@ button:hover {
 .has-background-rapids-image {
   background: url($header-bg-url) no-repeat;
   background-size: cover;
-  border-radius: 0.5rem;
-  border-top-right-radius: 0;
+  border-bottom-left-radius: 0.5rem;
```
Comment (bdice) [inline assets/scss/rapidsai/__general.scss:207]: Let’s define this in one line like `border-radius: 0 0 0.5rem 0.5rem;`

File: layouts/page/learn-more.html [hunk 20]
```diff
@@ -106,20 +106,30 @@ <h2><i class="fa-light fa-message-code"></i> Open Source </h2>
...
+          <h2><i class="fa-light fa-gauge-circle-bolt"></i> cuDF Pandas Accelerator </h2>
           <p>cuDF has a new pandas accelerator mode to speed up pandas workflows with zero code change. This is part of
             an on-going effort to streamline accelerated data science.
             <a href="/cudf-pandas">Learn More on the Launch Page <i class="fa-solid fa-arrow-up-right"></i> </a>
           </p>
         </div>
         <div class="col-md-6 py-3">
-          <h2><i class="fa-light fa-gauge-circle-bolt"></i> NetworkX Accelerator</h2>
+          <h2><i class="fa-light fa-gauge-circle-bolt"></i> Polars GPU Engine </h2>
+          <p>RAPIDS and Polars are collaborating to GPU accelerate Polars DataFrames, enabling workflows to utilize both the CPU and GPU to significantly speeding up workloads.
```
Comment (bdice) [inline layouts/page/learn-more.html:120]:
```suggestion
          <p>RAPIDS and Polars are collaborating to GPU accelerate Polars DataFrames, enabling workflows to utilize both the GPU and CPU to significantly speed up workloads.
```

File: layouts/page/learn-more.html [hunk 21]
```diff
@@ -106,20 +106,30 @@ <h2><i class="fa-light fa-message-code"></i> Open Source </h2>
...
+            <a href="/polars-gpu-engine">Learn More on the Launch Page <i class="fa-solid fa-arrow-up-right"></i> </a>
+          </p>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-md-6 py-3">
+          <h2><i class="fa-light fa-gauge-circle-bolt"></i> Network-X Accelerator</h2>
```
Comment (bdice) [inline layouts/page/learn-more.html:127]:
```suggestion
          <h2><i class="fa-light fa-gauge-circle-bolt"></i> NetworkX Accelerator</h2>
```

File: layouts/page/polars-gpu-engine.html [hunk 22]
```diff
@@ -0,0 +1,442 @@
...
+<!-- Intro -->
+<section class="row section mb-6">
+  <div class="col">
+    <div class="container">
+
+      <h1 class="text-center mb-10"> BRINGING ACCELERATED COMPUTING TO POLARS <br> [Open Beta] </h1>
+      <div class="row">
+        <div class="col-md-4 py-3">
+          <h2><i class="fa-light fa-file-binary"></i> Zero Code Change</h2>
+          <p>Pass <code>engine=”gpu”</code> to the Polars' collect API to add GPU acceleration to your data processing tasks like data aggregation, merge, window functions, and much more.</p>
```
Comment (bdice) [inline layouts/page/polars-gpu-engine.html:19]:
```suggestion
          <p>Pass <code>engine="gpu"</code> to the Polars <code>collect</code> API to add GPU acceleration to your data processing tasks like data aggregation, merges, window functions, and much more.</p>
```

File: layouts/page/polars-gpu-engine.html [hunk 23]
```diff
@@ -0,0 +1,442 @@
...
+  <div class="col">
+    <div class="container">
+      
+
+      <h1 class="mb-4">How To Use </h1>
+      <div class="row">
+        <div class="col">
+          <p>First, install the Polars GPU engine with a feature flag in your standard Python pip install command:</p>
+          <code class="code-sample mt-3">
+            pip install polars[gpu] —extra-index-url=https://pypi.nvidia.com
```
Comment (bdice) [inline layouts/page/polars-gpu-engine.html:88]: Some kind of autocorrect butchered this. Be careful with dashes and quotes.
```suggestion
            pip install polars[gpu] --extra-index-url=https://pypi.nvidia.com
```

File: layouts/page/polars-gpu-engine.html [hunk 24]
```diff
@@ -0,0 +1,442 @@
...
+            <br><br>
+           
+            ldf = pl.LazyFrame({
+                "a": [1.242, 1.535]
+            })
+           
+            <br><br>
+
+            gpu_engine = pl.GPUEngine(
+                device=0, #This is the default
```
Comment (bdice) [inline layouts/page/polars-gpu-engine.html:129]:
```suggestion
                device=0,  # This is the default
```

File: layouts/page/polars-gpu-engine.html [hunk 25]
```diff
@@ -0,0 +1,442 @@
...
+                raise_on_fail=True, #Fail loudly if can't execute on the GPU
```
Comment (bdice) [inline layouts/page/polars-gpu-engine.html:130]:
```suggestion
                raise_on_fail=True,  # Fail loudly if can't execute on the GPU
```

File: layouts/page/polars-gpu-engine.html [hunk 26]
```diff
@@ -0,0 +1,442 @@
...
+</div>
+
+<section class="row section bg-rapids-gray padding-top-slant padding-bottom-slant">
+  <div class="col">
+    <div class="container">
+
+      <h1 class="mb-4"> Frequently Asked Questions </h1>
+      <div class="row">
+
+        <div class="col ">
```
Comment (bdice) [inline layouts/page/polars-gpu-engine.html:160]:
```suggestion
        <div class="col">
```

File: layouts/page/polars-gpu-engine.html [hunk 27]
```diff
@@ -0,0 +1,442 @@
...
+            <div class="accordion-item">
+              <h2 class="accordion-header">
+                <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapse1" aria-expanded="true" aria-controls="collapse1">
+                  Do I have to install any RAPIDS specific package for the GPU engine to work? 
+                </button>
+              </h2>
+              <div id="collapse1" class="accordion-collapse collapse show" data-bs-parent="#faq1">
+                <div class="accordion-body">
+                  <p>No, you just need to install polars with a feature flag in your standard Python pip install command: <br>
+                    <code>pip install polars[gpu]--extra-index-url=https://pypi.nvidia.com</code>
```
Comment (bdice) [inline layouts/page/polars-gpu-engine.html:171]:
```suggestion
                    <code>pip install polars[gpu] --extra-index-url=https://pypi.nvidia.com</code>
```

File: layouts/page/polars-gpu-engine.html [hunk 28]
```diff
@@ -0,0 +1,442 @@
...
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <div class="accordion py-3" id="faq2">
+            <div class="accordion-item">
+              <h2 class="accordion-header">
+                <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapse2" aria-expanded="false" aria-controls="collapse2">
+                  Can I use Polars GPU engine on my local Windows machine?
```
Comment (bdice) [inline layouts/page/polars-gpu-engine.html:182]:
```suggestion
                  Can I use Polars GPU engine on a Windows machine?
```

File: layouts/page/polars-gpu-engine.html [hunk 29]
```diff
@@ -0,0 +1,442 @@
...
+                </button>
+              </h2>
+              <div id="collapse2" class="accordion-collapse collapse" data-bs-parent="#faq2">
+                <div class="accordion-body">
+                  <p>Yes, you can use the Polars GPU engine on a Windows machine by installing WSL2. Note: You would need WSL2 on a Windows 11 or above OS. For download and troubleshooting instructions, please visit <a href="https://docs.rapids.ai/install#wsl2" target="_blank">RAPIDS installation</a> page.</p>
```
Comment (bdice) [inline layouts/page/polars-gpu-engine.html:187]:
```suggestion
                  <p>Yes, you can use the Polars GPU engine on a Windows machine by installing WSL2 (requires Windows 11 or higher). For download and troubleshooting instructions, please visit the <a href="https://docs.rapids.ai/install#wsl2" target="_blank">RAPIDS installation</a> page.</p>
```

File: layouts/page/polars-gpu-engine.html [hunk 30]
```diff
@@ -0,0 +1,442 @@
...
+          <div class="accordion py-3" id="faq3">
+            <div class="accordion-item">
+              <h2 class="accordion-header">
+                <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapse3" aria-expanded="false" aria-controls="collapse3">
+                  Can I try the GPU engine with Polars' Rust API?
+                </button>
+              </h2>
+              <div id="collapse3" class="accordion-collapse collapse" data-bs-parent="#faq3">
+                <div class="accordion-body">
+                  <p>No, at the moment the GPU-based engine is only supported in Polars' python API.</p>
```
Comment (bdice) [inline layouts/page/polars-gpu-engine.html:202]:
```suggestion
                  <p>No, at the moment the GPU-based engine is only supported in Polars' Python API.</p>
```

File: layouts/page/polars-gpu-engine.html [hunk 31]
```diff
@@ -0,0 +1,442 @@
...
+          <div class="accordion py-3" id="faq4">
+            <div class="accordion-item">
+              <h2 class="accordion-header">
+                <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapse4" aria-expanded="false" aria-controls="collapse4">
+                  Are all operations within Polars supported?
+                </button>
+              </h2>
+              <div id="collapse4" class="accordion-collapse collapse " data-bs-parent="#faq4">
+                <div class="accordion-body">
+                  <p>The GPU engine currently supports most of the core expressions and data types, but not all. For a detailed list of supported expressions visit the <a href="https://docs.pola.rs/user-guide/gpu-support/" target="_blank">GPU Support page</a> on Polars website.</p>
```
Comment (bdice) [inline layouts/page/polars-gpu-engine.html:217]:
```suggestion
                  <p>The GPU engine currently supports most of the core expressions and data types, but not all. For a detailed list of supported expressions visit the <a href="https://docs.pola.rs/user-guide/gpu-support/" target="_blank">GPU Support page</a> on the Polars website.</p>
```

File: layouts/page/polars-gpu-engine.html [hunk 32]
```diff
@@ -0,0 +1,442 @@
...
+          <div class="accordion py-3" id="faq6">
+            <div class="accordion-item">
+              <h2 class="accordion-header">
+                <button class="accordion-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapse6" aria-expanded="false" aria-controls="collapse6">
+                  How can I provide feedback?
+                </button>
+              </h2>
+              <div id="collapse6" class="accordion-collapse collapse" data-bs-parent="#faq6">
+                <div class="accordion-body">
+                  <p>Raise an issue on <a href="https://github.com/pola-rs/polars/issues" target="_blank">Polars GitHub</a> and the project contributors will respond as soon as possible.</p>
```
Comment (bdice) [inline layouts/page/polars-gpu-engine.html:247]:
```suggestion
                  <p>Raise an issue on the <a href="https://github.com/pola-rs/polars/issues" target="_blank">Polars GitHub</a> and the project contributors will respond as soon as possible.</p>
```

File: layouts/page/polars-gpu-engine.html [hunk 33]
```diff
@@ -0,0 +1,442 @@
...
+</div>
+
+<section class="row section bg-rapids-gray padding-top-slant padding-bottom-slant">
+  <div class="col">
+    <div class="container">
+      <h1 class="mb-4">Accelerate Polars with RAPIDS </h1>
+      <div class="row">
+        <div class="col py-3">
+          <h2><i class="fa-light fa-cloud-arrow-down"></i> Try Now</h2>
+          <p>Try Polars GPU engine on <a href="https://colab.research.google.com/github/rapidsai-community/showcase/blob/main/accelerated_data_processing_examples/polars_gpu_engine_demo.ipynb" target="_blank">Google CoLab Right Now <i class="fa-solid fa-arrow-up-right"></i> </a></p>
```
Comment (bdice) [inline layouts/page/polars-gpu-engine.html:272]:
```suggestion
          <p>Try the Polars GPU engine on <a href="https://colab.research.google.com/github/rapidsai-community/showcase/blob/main/accelerated_data_processing_examples/polars_gpu_engine_demo.ipynb" target="_blank">Google Colab <i class="fa-solid fa-arrow-up-right"></i> </a></p>
```

File: layouts/page/polars-gpu-engine.html [hunk 34]
```diff
@@ -0,0 +1,442 @@
...
+        </div>
+        <div class="col py-3">
+          <h2><i class="fa-light fa-chalkboard-user"></i> Learn More</h2>
+          <p>Find out more on the <a href="https://docs.pola.rs/user-guide/gpu-support/" target="_blank">Polars GPU Support Page <i class="fa-solid fa-arrow-up-right"></i> </a></p>
+        </div>
+        <div class="col py-3">
+          <h2><i class="fa-light fa-code-merge"></i> Share Feedback</h2>
+          <p>Share feedback on the <a href="https://github.com/pola-rs/polars/issues" target="_blank">Polars GitHub Page <i class="fa-solid fa-arrow-up-right"></i> </a></p>
```
Comment (bdice) [inline layouts/page/polars-gpu-engine.html:280]:
```suggestion
          <p>Share feedback on the <a href="https://github.com/pola-rs/polars/issues" target="_blank">Polars GitHub repository <i class="fa-solid fa-arrow-up-right"></i> </a></p>
```

**Review by bdice — CHANGES_REQUESTED** [review]
First round of feedback.

**Review by bdice — APPROVED** [review]
I made a bunch of changes, and this is now good to go from my side.

**Pull request merged**